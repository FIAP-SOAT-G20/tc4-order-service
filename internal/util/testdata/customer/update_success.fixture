{
    "name": "John Doe 1 UPDATED",
    "email": "john.doe.1.updated@email.com"
}
