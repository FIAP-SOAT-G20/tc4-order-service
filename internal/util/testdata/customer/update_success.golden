{
    "id": 6,
    "name": "John Doe 6 UPDATED",
    "email": "john.doe.6.updated@email.com",
    "cpf": "000.000.000-06",
    "created_at": "2025-03-06T17:03:28Z",
    "updated_at": "2025-03-06T17:03:58Z"
}
